pkg/kp/rcstore/store.go

```diff
@@ -0,0 +1,32 @@
+package rcstore
+
+import (
+	"github.com/square/p2/pkg/labels"
+	"github.com/square/p2/pkg/pods"
+	"github.com/square/p2/pkg/rc/fields"
+)
+
+// Store represents an interface for persisting replication controllers to Consul,
+// as well as restoring replication controllers from Consul.
+type Store interface {
+	// Create creates a replication controller with the specified manifest and selectors.
+	// The node selector is used to determine what nodes the replication controller may schedule on.
+	// The pod label set is:
+	// 1) applied to every pod the replication controller schedules,
+	// 2) used (in addition to replication controller ID) to determine what pods are already scheduled.
```

Comment:
wait, this is an inaccurate comment. Currently I'm only using the replication controller ID to determine what pods are scheduled. Given that RC IDs are going to be UUIDs, that should uniquely identify pods that have been scheduled by this very same replication controller.

Is this the behavior we want, though? If I create a second replication controller that is identical to the first (same manifest, same node selector, same pod labels), *should* the latter see the pods scheduled by the former?

If the answer is yes, I'll change `currentNodes` to match.
If the answer is no, I need to remove this line as it's an inaccurate comment.